common-npm-packages/AzureRmDeploy-common-v2/package.json

```
@@ -1,6 +1,6 @@
 {
-    "name": "azure-pipelines-tasks-azurermdeploycommon",
-    "version": "2.198.2",
+    "name": "azure-pipelines-tasks-azurermdeploycommon-v2",
```

Maybe V3 would make a bit more sense since we're bumping major version to 3? :)
Up to you though
Same in other places too
```suggestion
    "name": "azure-pipelines-tasks-azurermdeploycommon-v3",
```

---

@DenisRumyantsev please update for other packages as well